tests/components/husqvarna_automower/test_sensor.py

```diff
@@ -74,6 +74,31 @@ async def test_cutting_blade_usage_time_sensor(
     assert state is None
 
 
+async def test_error_sensor(
+    hass: HomeAssistant,
+    mock_automower_client: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test error sensor."""
+    values = mower_list_to_dictionary_dataclass(
+        load_json_value_fixture("mower.json", DOMAIN)
+    )
+    await setup_integration(hass, mock_config_entry)
+
+    for state, expected_state in [
+        (None, "unknown"),
```

Comment:
Doesn't `None` mean there is no error rather than `unknown`?

Reply:
Yes, it means, that there is no error. And the python `None` is translated to `unknown` by Home Assistant. Should I use a custom string instead, like `no_error`?

Reply:
If `None` means no error it should return as "none"/"no_error"/... that can be translated into something that means no error.